### PR TITLE
Add spec URLs for all API features starting w/ “C”

### PR DESCRIPTION
**Pull request by sideshowbarker**

### CODE REVIEW

File: api/ClipboardItem.json [hunk 1]
```diff
@@ -203,6 +203,7 @@
       "presentationStyle": {
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/ClipboardItem/presentationStyle",
+          "spec_url": "https://w3c.github.io/clipboard-apis/#dom-clipboarditem-presentationstyle",
```
Comment (Elchi3) [inline api/ClipboardItem.json:206]: What's with the rest of this API?

File: api/Console.json [hunk 2]
```diff
@@ -214,6 +217,7 @@
       "countReset": {
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Console/countReset",
+          "spec_url": "https://console.spec.whatwg.org/#count",
```
Comment (Elchi3) [inline api/Console.json:220]:
```suggestion
          "spec_url": "https://console.spec.whatwg.org/#countreset",
```

File: api/CredentialsContainer.json [hunk 3]
```diff
@@ -50,6 +50,7 @@
       "create": {
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/CredentialsContainer/create",
+          "spec_url": "https://w3c.github.io/webappsec-credential-management/#dom-credentialscontainer-get",
```
Comment (Elchi3) [inline api/CredentialsContainer.json:53]:
```suggestion
          "spec_url": "https://w3c.github.io/webappsec-credential-management/#dom-credentialscontainer-create",
```

File: api/Crypto.json [hunk 4]
```diff
@@ -3,6 +3,7 @@
     "Crypto": {
       "__compat": {
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/Crypto",
+        "spec_url": "https://www.w3.org/TR/WebCryptoAPI/#crypto-interface",
```
Comment (Elchi3) [inline api/Crypto.json:6]:
```suggestion
        "spec_url": "https://w3c.github.io/webcrypto/#crypto-interface",
```

File: api/Crypto.json [hunk 5]
```diff
@@ -54,6 +55,7 @@
       "getRandomValues": {
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Crypto/getRandomValues",
+          "spec_url": "https://www.w3.org/TR/WebCryptoAPI/#Crypto-method-getRandomValues",
```
Comment (Elchi3) [inline api/Crypto.json:58]:
```suggestion
          "spec_url": "https://w3c.github.io/webcrypto/#Crypto-method-getRandomValues",
```

File: api/Crypto.json [hunk 6]
```diff
@@ -106,6 +108,7 @@
       "subtle": {
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Crypto/subtle",
+          "spec_url": "https://www.w3.org/TR/WebCryptoAPI/#dfn-Crypto",
```
Comment (Elchi3) [inline api/Crypto.json:111]:
```suggestion
          "spec_url": "https://w3c.github.io/webcrypto/#Crypto-attribute-subtle",
```

File: api/CryptoKey.json [hunk 7]
```diff
@@ -3,6 +3,7 @@
     "CryptoKey": {
       "__compat": {
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/CryptoKey",
+        "spec_url": "https://www.w3.org/TR/WebCryptoAPI/#dfn-CryptoKey",
```
Comment (Elchi3) [inline api/CryptoKey.json:6]:
```suggestion
        "spec_url": "https://w3c.github.io/webcrypto/#cryptokey-interface",
```

File: api/CryptoKey.json [hunk 8]
```diff
@@ -53,6 +54,7 @@
       "algorithm": {
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/CryptoKey",
+          "spec_url": "https://www.w3.org/TR/WebCryptoAPI/#dfn-CryptoKey",
```
Comment (Elchi3) [inline api/CryptoKey.json:57]:
```suggestion
          "spec_url": "https://w3c.github.io/webcrypto/#dom-cryptokey-algorithm",
```

File: api/CryptoKey.json [hunk 9]
```diff
@@ -104,6 +106,7 @@
       "extractable": {
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/CryptoKey",
+          "spec_url": "https://www.w3.org/TR/WebCryptoAPI/#dfn-CryptoKey",
```
Comment (Elchi3) [inline api/CryptoKey.json:109]:
```suggestion
          "spec_url": "https://w3c.github.io/webcrypto/#dom-cryptokey-extractable",
```

File: api/CryptoKey.json [hunk 10]
```diff
@@ -206,6 +209,7 @@
       "type": {
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/CryptoKey",
+          "spec_url": "https://www.w3.org/TR/WebCryptoAPI/#dfn-CryptoKey",
```
Comment (Elchi3) [inline api/CryptoKey.json:212]:
```suggestion
          "spec_url": "https://w3c.github.io/webcrypto/#dom-cryptokey-type",
```

File: api/CryptoKey.json [hunk 11]
```diff
@@ -257,6 +261,7 @@
       "usages": {
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/CryptoKey",
+          "spec_url": "https://www.w3.org/TR/WebCryptoAPI/#dfn-CryptoKey",
```
Comment (Elchi3) [inline api/CryptoKey.json:264]:
```suggestion
          "spec_url": "https://w3c.github.io/webcrypto/#dom-cryptokey-usages",
```

File: api/CryptoKeyPair.json [hunk 12]
```diff
@@ -3,6 +3,7 @@
     "CryptoKeyPair": {
       "__compat": {
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/CryptoKeyPair",
+        "spec_url": "https://www.w3.org/TR/WebCryptoAPI/#dfn-CryptoKeyPair",
```
Comment (Elchi3) [inline api/CryptoKeyPair.json:6]:
```suggestion
        "spec_url": "https://w3c.github.io/webcrypto/#keypair",
```

**Review by Elchi3 — CHANGES_REQUESTED** [review]

File: api/ClipboardItem.json (hunk 1 above)
Comment (sideshowbarker) [reply api/ClipboardItem.json:206]: > What's with the rest of this API?

`presentationStyle` is the only member that has an actual MDN article. The object itself and all other other members aren’t documented in MDN yet.

Comment (Elchi3) [reply api/ClipboardItem.json:206]: hm, but there are pages:
https://developer.mozilla.org/en-US/docs/Web/API/ClipboardItem
https://developer.mozilla.org/en-US/docs/Web/API/ClipboardItem/ClipboardItem
https://developer.mozilla.org/en-US/docs/Web/API/ClipboardItem/getType
https://developer.mozilla.org/en-US/docs/Web/API/ClipboardItem/types

Comment (sideshowbarker) [reply api/ClipboardItem.json:209]: > hm, but there are pages:
> [developer.mozilla.org/en-US/docs/Web/API/ClipboardItem](https://developer.mozilla.org/en-US/docs/Web/API/ClipboardItem)
> [developer.mozilla.org/en-US/docs/Web/API/ClipboardItem/ClipboardItem](https://developer.mozilla.org/en-US/docs/Web/API/ClipboardItem/ClipboardItem)
> [developer.mozilla.org/en-US/docs/Web/API/ClipboardItem/getType](https://developer.mozilla.org/en-US/docs/Web/API/ClipboardItem/getType)
> [developer.mozilla.org/en-US/docs/Web/API/ClipboardItem/types](https://developer.mozilla.org/en-US/docs/Web/API/ClipboardItem/types)

Indeed yeah — thanks for catching those. The cause was that the script I run for scraping the spec URLs wasn’t catching pages that had been added relatively recently (the last ~3 months or so). But  I’ve fixed it now and pushed an additional commit here — which, along with adding the ClipboardItem spec URLs, also adds them for CharacterData, ContactsManager, and CookieChangeEvent.

(And I’ll also go back now and see what other pages got added recently that the script missed — and then raise other PRs as needed for those.)

Comment (Elchi3) [reply api/ClipboardItem.json:209]: Great! Glad we spotted this before going further into the world of API docs. Happy to review PRs for those missed spec_urls

**Review by Elchi3 — APPROVED** [review]

**Pull request merged**